validator/remote/src/main/java/tech/pegasys/teku/validator/remote/typedef/ValidatorRestApiClient.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.validator.remote.typedef;
+
+import java.util.Optional;
+import tech.pegasys.teku.spec.datastructures.genesis.GenesisData;
+
+public interface ValidatorRestApiClient {
```

Comment:
Do we need this interface? I can see it would be helpful if we're feature toggling between the two implementations but if we're just doing a migration we probably don't need it?

Reply:
true, the way its structured this could just be deleted, I guess i was mirroring the other apiclient...

Reply:
deleted interface